lighthouse-core/gather/gather-runner.js

```diff
@@ -29,8 +29,22 @@ const UIStrings = {
    */
   warningTimeout: 'The page loaded too slowly to finish within the time limit. ' +
   'Results may be incomplete.',
+  /**
+   * @description Warning that the host device where Lighthouse is running appears to have a slower
+   * CPU than the expected Lighthouse baseline.
+   */
+  warningSlowHostCpu: 'The device that ran this test appears to have a slower CPU than  ' +
+  'Lighthouse expects. This can negatively affect your performance score. Learn more about using ' +
+  '[custom throttling settings](https://github.com/GoogleChrome/lighthouse/blob/ccbc8002fd058770d14e372a8301cc4f7d256414/docs/throttling.md#calibrating-multipliers) ' +
+  'to calibrate your device.',
 };
 
+/**
+ * Warn when the CPU seemed to be at least ~2x weaker than our regular target device.
+ * @see https://github.com/GoogleChrome/lighthouse/blob/ccbc8002fd058770d14e372a8301cc4f7d256414/docs/throttling.md#calibrating-multipliers
+ */
+const SLOW_CPU_BENCHMARK_INDEX_THRESHOLD = 1200;
```

Comment:
set at the higher end of mobile and in the middle of "low-end" desktop, in the devices tested the ones scoring ~1200 required an additional 2x multiplier compared to a device scoring at ~1700

it's just a warning, not automatic, so I think this is a good start. Probably won't capture the impact of a bunch of busy apps hogging CPU on a high-end laptop, but should be enough to flag a really low end netbook

Reply:
This would include... <checks notes>.... 99% of LR runs.  (But they're excluded because this is only for CLI channel, phew)

I'm a little scared it's still high even for CLI.

do we still have that test page online ?

Reply:
Honestly, that tracks. LR is pretty [dang weak](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=http%3A%2F%2Fmelodic-class.glitch.me%2Fbenchmark-quick.html).


Local JS execution time
![image](https://user-images.githubusercontent.com/2301202/91920924-e2563380-ec8f-11ea-94bb-a2423796f0bc.png)


LR JS execution time
![image](https://user-images.githubusercontent.com/2301202/91920907-dbc7bc00-ec8f-11ea-8787-dd4d7e418b41.png)


This is the new benchmark test page that's quick http://melodic-class.glitch.me/benchmark-quick.html